src/robusta/core/playbooks/internal/ai_integration.py

```diff
@@ -243,3 +249,100 @@ def delayed_health_check(event: KubernetesAnyChangeEvent, action_params: Delayed
+        holmes_req = HolmesIssueChatRequest(
+            ask=params.ask,
+            conversation_history=params.conversation_history,
+            investigation_result=params.context.investigation_result,
+            issue_type=params.context.issue_type,
+        )
+        result = requests.post(f"{holmes_url}/api/issue_chat", data=holmes_req.json())
+        result.raise_for_status()
+
+        holmes_result = HolmesChatResult(**json.loads(result.text))
+
+        finding = Finding(
+            title=f"AI Analysis of {conversation_title}",
+            aggregation_key="HolmesConversationResult",
+            subject=FindingSubject(
+                name=params.resource.name if params.resource else "",
+                namespace=params.resource.namespace if params.resource else "",
+                subject_type=FindingSubjectType.from_kind(params_resource_kind)
+                if params.resource
+                else FindingSubjectType.TYPE_NONE,
+                node=params.resource.node if params.resource else "",
+                container=params.resource.container if params.resource else "",
+            ),
+            finding_type=FindingType.AI_ANALYSIS,
+            failure=False,
+        )
+        finding.add_enrichment(
+            [HolmesChatResultsBlock(holmes_result=holmes_result)], enrichment_type=EnrichmentType.ai_analysis
+        )
+
+        event.add_finding(finding)
+
+    except Exception as e:
+        logging.exception(f"Failed to get holmes chat for {conversation_title}", exc_info=True)
+        if isinstance(e, requests.ConnectionError):
+            raise ActionException(ErrorCodes.HOLMES_CONNECTION_ERROR, "Holmes endpoint is currently unreachable.")
+        elif isinstance(e, requests.HTTPError):
+            if e.response.status_code == 401 and "invalid_api_key" in e.response.text:
+                raise ActionException(ErrorCodes.HOLMES_REQUEST_ERROR, "Holmes invalid api key.")
+
+            raise ActionException(ErrorCodes.HOLMES_REQUEST_ERROR, "Holmes internal configuration error.")
+        else:
+            raise ActionException(ErrorCodes.HOLMES_UNEXPECTED_ERROR, "An unexpected error occured.")
+
+
+@action
+def holmes_chat(event: ExecutionBaseEvent, params: HolmesChatParams):
+    holmes_url = HolmesDiscovery.find_holmes_url(params.holmes_url)
+    if not holmes_url:
+        raise ActionException(ErrorCodes.HOLMES_DISCOVERY_FAILED, "Robusta couldn't connect to the Holmes client.")
+
+    cluster_name = event.get_context().cluster_name
+    account_id = event.get_context().account_id
+
+    try:
+        holmes_req = HolmesChatRequest(ask=params.ask, conversation_history=params.conversation_history)
+        result = requests.post(f"{holmes_url}/api/chat", data=holmes_req.json())
+        result.raise_for_status()
+        holmes_result = HolmesChatResult(**json.loads(result.text))
+
+        finding = Finding(
+            title=f"AI Ask Chat for {cluster_name} cluster {account_id} account_id",
```

Comment:
I don't think we want to add the account id, it's just a uuid
I think the cluster is probably redundant as well